lib/main.dart

```diff
@@ -1,16 +1,19 @@
 import 'package:flutter/material.dart';
 import 'package:vaultSDK/udContext.dart';
+import 'package:vaultSDK/udManager.dart';
 
 import 'auth/login.dart';
 import 'auth/home.dart';
 import 'auth/settings.dart';
+import 'auth/SceneViewer.dart';
```

Comment:
might be case sensitive. 
```suggestion
import 'auth/sceneViewer.dart';
```